docs/SDK-GOVERNANCE.md

```diff
@@ -10,6 +10,23 @@ The community is organized as follows:
   [cloudevents/sdk-go project](https://github.com/cloudevents/sdk-go)
 - The union of all the `sdk-*-maintainers` assembles the _sdk maintainers_ group
 
+## SDK Repository Requirements
+
+Each SDK must include the following:
+- a pointer to it's github repo from the CloudEvents [README.md](README.md)
```

Comment:
it's => its

Reply:
OMG I can't type that correctly! Seriously, I do know it!